configs/bs-en-spring-2024.yml

```diff
@@ -0,0 +1,114 @@
+  - opus_wikimedia/v20230407 #                               28,167 sentences
+  - opus_QED/v2.0a #                                         12,541 sentences
+  - opus_TED2020/v1 #                                        11,638 sentences
+  - opus_NeuLab-TedTalks/v1 #                                 6,136 sentences
+  - opus_EUbookshop/v2 #                                        558 sentences
+  - opus_Tatoeba/v2023-04-12 #                                  515 sentences
+  - opus_tldr-pages/v2023-08-29 #                               479 sentences
+  - opus_ELRC-3047-wikipedia_health/v1 #                        205 sentences
+  - opus_ELRC-wikipedia_health/v1 #                             205 sentences
+  - opus_ELRC_2922/v1 #                                         204 sentences
+
+  # The monolingual data contains:
+  #   ~9,416,145 sentences
+  mono-src:
+  - news-crawl_news.2018  #              ~9,748 sentences (1.1 MB)
+  - news-crawl_news.2019 #            ~965,035 sentences (109.0 MB)
+  - news-crawl_news.2020 #          ~1,821,799 sentences (205.9 MB)
+  - news-crawl_news.2021 #          ~2,178,346 sentences (246.2 MB)
+  - news-crawl_news.2022 #          ~2,233,374 sentences (252.4 MB)
+  - news-crawl_news.2023 #          ~2,207,843 sentences (249.5 MB)
+
+  # The monolingual data contains:
+  #   ~209,074,237 sentences
+  mono-trg:
+  - news-crawl_news.2007  #          ~1,630,834 sentences (184.3 MB)
+  - news-crawl_news.2008 #          ~5,648,654 sentences (638.3 MB)
+  - news-crawl_news.2009 #          ~6,879,522 sentences (777.4 MB)
+  - news-crawl_news.2010 #          ~3,406,380 sentences (384.9 MB)
+  - news-crawl_news.2011 #          ~6,628,308 sentences (749.0 MB)
+  - news-crawl_news.2012 #          ~6,715,609 sentences (758.9 MB)
+  - news-crawl_news.2013 #         ~11,050,614 sentences (1.2 GB)
+  - news-crawl_news.2014 #         ~11,026,051 sentences (1.2 GB)
+  - news-crawl_news.2015 #         ~11,182,484 sentences (1.3 GB)
+  - news-crawl_news.2016 #          ~8,366,518 sentences (945.4 MB)
+  - news-crawl_news.2017 #         ~12,276,499 sentences (1.4 GB)
+  - news-crawl_news.2018 #          ~8,303,432 sentences (938.3 MB)
+  - news-crawl_news.2019 #         ~19,386,668 sentences (2.2 GB)
+  - news-crawl_news.2020 #         ~24,070,652 sentences (2.7 GB)
+  - news-crawl_news.2021 #         ~23,139,914 sentences (2.6 GB)
+  - news-crawl_news.2022 #         ~24,900,055 sentences (2.8 GB)
+  - news-crawl_news.2023 #         ~24,462,043 sentences (2.8 GB)
+marian-args:
+  decoding-backward:
+    beam-size: '12'
+    mini-batch-words: '2000'
+  decoding-teacher:
+    mini-batch-words: '4000'
+    precision: float16
+  training-backward:
+    early-stopping: '5'
+  training-teacher:
+    early-stopping: '20'
+  training-student:
+    early-stopping: '20'
+  training-student-finetuned:
+    early-stopping: '20'
+target-stage: all
+wandb-publication: true
+taskcluster:
+  split-chunks: 20
+  worker-classes:
+    default: gcp-spot
```

Comment:
Like we discussed with @bhearsum we might want to switch bicleaner to gcp-standard if a single parallel dataset is too large to prevent restarts on preemptions that will slow down our training significantly.

Reply:
a large nllb dataset seems to be the case only for en-ru so let's see how it performs first